### ISSUE TITLE
Browser Support

### ISSUE THREAD

**Issue by joeskeen** [HealthCatalyst/Fabric.Cashmere#19]
## tl;dr

Browser support needs to be decided upon and documented.  Currently there is some inconsistencies about what we support.

`tsconfig.json`:

```json
...
"lib": [
  "es2016",
  "dom"
],
...
```

This suggests that we are targeting only browsers supporting ES2016, or that we are including polyfills for any browsers that don't support ES2016.

`package.json`

```json
...
"dependencies": {
...
  "core-js": "^2.4.1",
...
```

As [core-js](https://github.com/zloirock/core-js) is a polyfill library, it seems that this is OK.  But I also heard discussion on #18 about using Lodash in addition to / instead of polyfills, and when I run `npm run package`, the process fails due to TSC error: 

```
lib/button/button.component.ts(35,33): error TS2339: Property 'includes' does not exist on type 'string[]'.
```
(`Array.includes` was added in ES2016)

So what is our official browser support, and what is our strategy on polyfills?

**Comment by benjanderson** [comment]
This was fixed by adding polyfills, although they were not added through the rollup build. As these are well established polyfills, I feel we can have the standard of "we support all modern browsers" and each of the consumers of the library can add those polyfills. 

We can either document what polyfills to include or as it is such a common thing we could leave it up to the consumer and close this issue.

**Comment by joeskeen** [comment]
Agreed, we should add that to the Readme then close this issue.

**Comment by jtrujill** [comment]
Currently we support IE11 and up according to http://cashmere.healthcatalyst.net/guides/supported-browsers. We just need to make a note of this in the readme
